lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -34,13 +35,12 @@ class SqliteStore(BaseStore):
     def __init__(self, db_location, loop=None, sid_digits=2):
         self._sid_digits = sid_digits  # last digits to be included from subscriber id
         self._n_shards = 10**sid_digits
-        self._db_locations = self._create_db_locations(
-            db_location, self._n_shards,
-        )
+        self._db_locations, self._digest_db_location = \
+            self._create_db_locations(db_location, self._n_shards)
         self._create_store()
         self._on_ready = OnDataReady(loop=loop)
 
-    def _create_db_locations(self, db_location, n_shards):
+    def _create_db_locations(self, db_location, n_shards) -> [list, str]:
```

Comment:
Go ahead and annotate the parameter types as well. Basically, in Python, any time you update an untyped fn signature, go ahead and type the entire fn signature